tutorial1_intro/README.md

```diff
@@ -0,0 +1,23 @@
+# Tutorial 1
+
+This introductory tutorial will focus primarily on helping you get your environment set up for future code development. We will also walk through an example that (a) loads in a new dataset and (b) uses D3.js to render a simple HTML table to the browser.
+
+The goals for this tutorial are:
+
+- for students to get comfortable with the [Github workflow](../GIT_SETUP.md), and with using their [local development environment](../README.md).
+- for students to understand how to load in a dataset using [d3-fetch](https://github.com/d3/d3-fetch) (d3.json, d3.csv, etc.).
+- to introduce [d3-selections](https://bost.ocks.org/mike/selection/) and [d3 data binding](https://observablehq.com/@d3/selection-join).
+- to give the tools to create a simple [HTML table](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table).
+
+## Assignment:
+
+1. Go through the [demo code](demo/) understand how the code works
+2. Replicate the process for **your own dataset** in the files present in the root of this directory ([index.html](index.html), [style.css](style.css), [main.js](main.js))
+3. Use the Github workflow to deploy your work to **your fork** of the course repository
+4. Post the link to your deployed example on the Commons. **//TODO: check this @Ellie**
```

Comment:
@efrymire - where should they submit their link?

Also - re:L14. My thought is that maybe for the first week their assignment is just to implement our demo example in their new environment with their own dataset. What do you think?

Reply:
Ideally through some sort of submission system (I guess... commons? blackboard?) but if we haven't gotten that set up by thursday lets just have them email it